src/locale/pages/legacy/en.yml

```diff
@@ -0,0 +1,6 @@
+continueButton: Continue to old Zetkin
+header: You are being redirected
+info: |
+  You are using a prerelease version of Zetkin and the link you clicked has
+  not yet been implemented. You are being redirected to the equivalent page
+  in an older version of Zetkin.
```

Comment:
> You are using a prerelease version of Zetkin which doesn't support the feature you want to use. You are being redirected to the older version of Zetkin which supports that feature.

What do you think about that text?

Reply:
Yeah, it's better! I'm going to change it. 😊